src/Uno.UI/UI/Xaml/UIElement.wasm.cs

```diff
@@ -729,5 +733,25 @@ private static IEnumerable<string> GetClassesForType(Type type)
 				}
 			}
 		}
+
+		private InputCursor _protectedCursor = InputSystemCursor.Create(InputSystemCursorShape.Arrow);
+
+		protected InputCursor ProtectedCursor
+		{
+			get => _protectedCursor;
+			set
+			{
+				_protectedCursor = value;
+				Internal_SetProtectedCursor();
+			}
+
+		}
+
+		private void Internal_SetProtectedCursor()
+		{
+			var inputSystemCursor = _protectedCursor as InputSystemCursor;
+			var cursorShape = inputSystemCursor.CursorShape.ToCssProtectedCursor();
+			this.SetStyle("cursor", cursorShape);
```

Comment:
This can crash with null reference exception if ProtectedCursor is set to an instance of `InputDesktopResourceCursor`. I'd do it like this:

```suggestion
			if (_protectedCursor is InputSystemCursor inputSystemCursor)
			{
				var cursorShape = inputSystemCursor.CursorShape.ToCssProtectedCursor();
				this.SetStyle("cursor", cursorShape);
			}
```

Reply:
```suggestion
			if (_protectedCursor is InputSystemCursor inputSystemCursor)
			{
				var cursorShape = inputSystemCursor.CursorShape.ToCssProtectedCursor();
				this.SetStyle("cursor", cursorShape);
			}
			else
			{
				this.ResetStyle("cursor");
			}
```

You must reset the value is the user sets `null` for the `ProtectedCursor`.